src/components/Utilities/Spacing.css

```diff
@@ -84,6 +84,18 @@
   margin-top: var(--eds-size-4) !important;
 }
 
+/**
+ * Margin top xl mobile
+ * 1) Forces margin top to 2rem (on smaller viewports only)
+ */
+.u-margin-top-xl-mobile {
+  margin-top: var(--eds-size-4) !important;
+
+  @media all and (min-width: $eds-bp-md) {
+    margin-top: unset !important;
+  }
+}
```

Comment:
Could you help me understand this `unset` stuff? Why can't we just target the size that needs the margin? Like
```
.u-margin-top-xl-mobile {
  @media all and (max-width: $eds-bp-md) {
    margin-top: var(--eds-size-4) !important;
  }
}
```
Same for the `u-mobile-hidden` class.

Reply:
Great question! Because the style on `.page-header` changes `flex-direction` from `column` to `row` using the media query `min-width: $eds-bp-md`. So any styles that are affecting the same elements at the same breakpoint should use the same media query (i.e., all use min-width OR all use max-width). Otherwise there's a 1px "gap" where the min-width rule is being applied but the max-width rule isn't. 

https://user-images.githubusercontent.com/24812780/181619408-4bf7f135-9065-4a3c-801e-bf0ae28defd8.mov

In a perfect world, all media queries in a project would either be "max-width" or "min-width"; if so, there's never any gap between where one rule starts and another stops.

Reply:
BTW you can see the "gap" between media queries on that same page with the header size on "Select projects for your History 6 plan". Out of the scope of this ticket though, whew.

Reply:
Oo nice catch. Could you leave a small comment about needing to use the same media query as the `.page-header`?